translators/test/snapshot/Pattern_Matching_Datatypes.idr

```diff
@@ -8,7 +8,7 @@ data D : Type where
  C5 : D
 
 
-F : D -> Nat
+F : (C : D) -> Nat
```

Comment:
Do we want to be adding a telescope cell here?

Reply:
We don't really, but the encoding in the test case has the `C` so the printer doesn't have enough information to throw it out. The issue is the Rocq needs this information, so it bleeds into the other tests.